### PR TITLE
06.06 Changelog

### PR DESCRIPTION
**Pull request by Cahllagerfeld**
Adds a new Changelog

<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2184"></a>

<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2184"></a>

### CODE REVIEW

File: src/lib/contents/changelog/2022-06-06.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+- [#10302](https://github.com/gitpod-io/gitpod/pull/10302) - [dashboard] Introduce an experimental usage-based billing feature flag (off by default) <Contributors usernames="easyCZ,jankeromnes" />
+- [#10182](https://github.com/gitpod-io/gitpod/pull/10182) - Fix, polish, and release new simplified Team Billing <Contributors usernames="geropl,gtsiolis,jankeromnes" />
+- [#10343](https://github.com/gitpod-io/gitpod/pull/10343) - [installer] Command line flag to configure strict config parsing <Contributors usernames="corneliusludmann,easyCZ,geropl" />
+- [#10364](https://github.com/gitpod-io/gitpod/pull/10364) - [content-service] Improve restoration of extended attributes <Contributors usernames="aledbf,utam0k" />
+- [#10290](https://github.com/gitpod-io/gitpod/pull/10290) - [experimental] add a metric to track volume snapshot time <Contributors usernames="Furisto,jenting,sagor999" />
+- [#10354](https://github.com/gitpod-io/gitpod/pull/10354) - [kots]: correctly escape params in installation configmap <Contributors usernames="MrSimonEmms,corneliusludmann" />
```
Comment (atduarte) [inline src/lib/contents/changelog/2022-06-06.md:62]: Remove

File: src/lib/contents/changelog/2022-06-06.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+- [#10177](https://github.com/gitpod-io/gitpod/pull/10177) - JetBrains Gateway: The "Connect" button now gets disabled while a JetBrains Client is connected to the workspace. <Contributors usernames="akosyakov,felladrin,iQQBot,yaohui-wyh" />
+- [#10339](https://github.com/gitpod-io/gitpod/pull/10339) - [ws-daemon] Use a metric for backup waiting time instead of logs <Contributors usernames="aledbf,csweichel" />
```
Comment (atduarte) [inline src/lib/contents/changelog/2022-06-06.md:64]: Remove

**Review by atduarte — CHANGES_REQUESTED** [review]
Suggestions made in line with https://gitpod.slack.com/archives/C01KGM9AW4W/p1654677957766019

File: src/lib/contents/changelog/2022-06-06.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+TODO: Insert the featured changelog entry here.
+
+<p><Contributors usernames="" /></p>
+
+### Roadmap updates
+
+<div class="mt-medium">
+
+**Item1** - Roadmap issue: [#XXXX](https://github.com/gitpod-io/gitpod/issues/XXXX) <Badge text="alpha" variant="pink" class="ml-1.5 bg-salmon" />
+
```
Comment (loujaybee) [inline src/lib/contents/changelog/2022-06-06.md:23]: **Dotfiles** - Roadmap issue: [#5198](https://github.com/gitpod-io/gitpod/issues/5198) <Badge text="alpha" variant="pink" class="ml-1.5 bg-salmon" />

- [#10398](https://github.com/gitpod-io/gitpod/pull/10398) - Support for dotfiles is now generally available <Contributors usernames="easyCZ,gtsiolis,loujaybee" />

File: src/lib/contents/changelog/2022-06-06.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+</div>
+
+### Fixes and improvements
+
+- [#10469](https://github.com/gitpod-io/gitpod/pull/10469) - Check docker-compose download <Contributors usernames="aledbf,sagor999" />
+- [#10458](https://github.com/gitpod-io/gitpod/pull/10458) - Update docker compose to v2.6.0 <Contributors usernames="aledbf,kylos101,sagor999" />
+- [#10442](https://github.com/gitpod-io/gitpod/pull/10442) - Revert "[baseserver] Change default metrics port to 9502 to not clash with kube-rbac-proxy" <Contributors usernames="aledbf,kylos101" />
+- [#10414](https://github.com/gitpod-io/gitpod/pull/10414) - Improve workspace failure error messages <Contributors usernames="csweichel,sagor999" />
+- [#10413](https://github.com/gitpod-io/gitpod/pull/10413) - [supervisor] improve error message around user group and uid <Contributors usernames="iQQBot,sagor999" />
+- [#10398](https://github.com/gitpod-io/gitpod/pull/10398) - Support for dotfiles has moved out of beta and is now generally available <Contributors usernames="easyCZ,gtsiolis,loujaybee" />
```
Comment (loujaybee) [inline src/lib/contents/changelog/2022-06-06.md:33]: Moved to feature

File: src/lib/contents/changelog/2022-06-06.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+- [#10395](https://github.com/gitpod-io/gitpod/pull/10395) - ws-daemon: Apply the xfs limit in stages. <Contributors usernames="jenting,kylos101,utam0k" />
+- [#10388](https://github.com/gitpod-io/gitpod/pull/10388) - Gitpod CLI has a new command to list the ports from the workspace: gp ports list <Contributors usernames="andreafalzetti,felladrin,mustard-mh" />
```
Comment (loujaybee) [inline src/lib/contents/changelog/2022-06-06.md:35]: Added above. 

```suggestion
```

File: src/lib/contents/changelog/2022-06-06.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+- [#10394](https://github.com/gitpod-io/gitpod/pull/10394) - dashboard: single quote connect via ssh connection string <Contributors usernames="easyCZ,ghuntley" />
+- [#10229](https://github.com/gitpod-io/gitpod/pull/10229) - Change `/workspace` files default permissions from 0750 to 0755 <Contributors usernames="kylos101,mustard-mh,sagor999" />
+- [#10107](https://github.com/gitpod-io/gitpod/pull/10107) - [JetBrains] Show notification when port becomes available <Contributors usernames="akosyakov,andreafalzetti,mustard-mh" />
```
Comment (loujaybee) [inline src/lib/contents/changelog/2022-06-06.md:38]: Mentioned above 

```suggestion
```

File: src/lib/contents/changelog/2022-06-06.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+- [#10343](https://github.com/gitpod-io/gitpod/pull/10343) - [installer] Command line flag to configure strict config parsing <Contributors usernames="corneliusludmann,easyCZ,geropl" />
+- [#10364](https://github.com/gitpod-io/gitpod/pull/10364) - [content-service] Improve restoration of extended attributes <Contributors usernames="aledbf,utam0k" />
+- [#10290](https://github.com/gitpod-io/gitpod/pull/10290) - [experimental] add a metric to track volume snapshot time <Contributors usernames="Furisto,jenting,sagor999" />
+- [#10177](https://github.com/gitpod-io/gitpod/pull/10177) - JetBrains Gateway: The "Connect" button now gets disabled while a JetBrains Client is connected to the workspace. <Contributors usernames="akosyakov,felladrin,iQQBot,yaohui-wyh" />
+- [#10280](https://github.com/gitpod-io/gitpod/pull/10280) - [initializer] Fix issue with publicly signed SCM's on a self-signed Gitpod instance <Contributors usernames="Pothulapati,corneliusludmann,csweichel" />
+- [#10291](https://github.com/gitpod-io/gitpod/pull/10291) - SSH Gateway support remote port forward <Contributors usernames="akosyakov,iQQBot,mustard-mh,sagor999" />
```
Comment (loujaybee) [inline src/lib/contents/changelog/2022-06-06.md:51]: Moved above. 

```suggestion
```

**Comment by gtsiolis** [comment]
@Cahllagerfeld I've updated the header image per [relevant discussion](https://gitpod.slack.com/archives/C022HAT0H5Z/p1654705726700599?thread_ts=1654582691.284399&cid=C022HAT0H5Z) (internal).

| BEFORE | AFTER |
|-|-|
| 3000-gitpodio-website-s5j1f8zvej0 ws-eu47 gitpod io_changelog | 3000-gitpodio-website-5u7tmxjumst ws-eu47 gitpod io_changelog |

**Pull request merged**